src/App.js

```diff
@@ -1,19 +1,21 @@
-import styled from 'styled-components/macro'
-import locations from './data.json'
-import LocationsPage from './pages/LocationsPage'
+import { useEffect, useState } from 'react'
+import AddGamePage from './pages/AddGamePage'
+import { setLocalStorage, getLocalStorage } from './utils/localStorage'
 
 function App() {
+  const [games, setGames] = useState(getLocalStorage('games') ?? [])
+  useEffect(() => {
+    setLocalStorage('games', games)
+  }, [games])
   return (
-    <AppWrapper>
-      <LocationsPage locations={locations} />
-    </AppWrapper>
+    <div>
+      <AddGamePage games={games} onSubmit={handleSelected} />
+    </div>
```

Comment:
```suggestion
      <AddGamePage games={games} onSubmit={handleSelected} />
```
I don't think you need the `div` here

Reply:
true! 😅 thank you :)